collectors/python.d.plugin/python.d.conf

```diff
@@ -72,6 +72,7 @@ logind: no
 nginx_log: no
 # nsd: yes
 # ntpd: yes
+openldap: no
```

Comment:
please change to `# openldap: yes`

The module will attempt to connect only on servers with `python-ldap` package installed.

Reply:
done